src/adapter/driver/api/docs/paths/cliente.ts

```diff
@@ -0,0 +1,171 @@
+import { serverError, notFound, unprocessableEntity } from "../defaults";
+
+export const ClientePaths = {
+    "/cliente/by-email/{email}": {
+        get: {
+            tags: ["cliente"],
+            summary: "Rota para encontrar um cliente por email",
+            parameters: [
+                {
+                    in: "path",
+                    name: "email",
+                    description: "email do cliente a ser encontrado",
+                    required: true,
+                    schema: {
+                        type: "string",
+                    },
+                },
+            ],
+            responses: {
+                200: {
+                    description: "Cliente encontrado",
+                    content: {
+                        "application/json": {
+                            schema: {
+                                type: "object",
+                                properties: {
+                                    id: {
+                                        type: "string",
+                                    },
+                                    nome: {
+                                        type: "string",
+                                    },
+                                    email: {
+                                        type: "string",
+                                    },
+                                    cpf: {
+                                        type: "string",
+                                    },
+                                },
+                                required: ["nome", "email"],
+                            },
+                        },
+                    },
+                },
+                404: {
+                    ...notFound,
+                },
+                422: {
+                    ...unprocessableEntity,
+                },
+                500: {
+                    ...serverError,
+                },
+            },
+        },
+    },
+    "/cliente/by-cpf/{cpf}": {
+        get: {
+            tags: ["cliente"],
+            summary: "Rota para encontrar um cliente por cpf",
+            parameters: [
+                {
+                    in: "path",
+                    name: "cpf",
+                    description: "cpf do cliente a ser encontrado",
```

Comment:
Sugestão, aqui eu adicionaria o formato que estamos esperando o CPF. Pq eu fui testar colocando só os números direto e tomei um erro, pq não observei que esperávamos o número formatado.
```suggestion
                    description: "cpf do cliente a ser encontrado (ex - 000.000.000-00)",
```

Reply:
- Adicionei na opção de `exemples` do swagger e na mensagem de erro de validação 👍
- Coloquei um site para gerar o cpf de teste também